internal/local/app_test.go

```diff
@@ -771,3 +748,19 @@ var appData20210101Remote = AppDataV2{AppStructureV2{
 	DeploymentModel:       realm.DeploymentModelGlobal,
 	AllowedRequestOrigins: []string{"http://localhost:8080"},
 }}
+
+var appData20210101Nested = AppDataV2{AppStructureV2{
+	ConfigVersion:         realm.AppConfigVersion20210101,
+	ID:                    "20210101-nested-abcde",
+	Name:                  "20210101-nested",
+	Location:              realm.LocationVirginia,
+	DeploymentModel:       realm.DeploymentModelGlobal,
+	AllowedRequestOrigins: []string{"http://localhost:8080"},
+	GraphQL: GraphQLStructure{
+		Config: map[string]interface{}{
+			"use_natural_pluralization": true,
+		},
+	},
+}}
+
+var appData20210101Invalid = AppDataV2{AppStructureV2{}}
```

Comment:
[ocd/nit] this doesn't seem to have a similar structure as the other "invalid" cases, is that intentional or just convenience?  might be worth either building this out to match, or adding a `if name == "invalid"` short-circuit near the top of the `createAppDataV1` helper

Reply:
we could actually remove the `appData****Invalid` vars since the apps are not expected to be found (because config file doesn't contain a valid version)
Since the apps aren't found, there is no appData to compare theses vars with.